src/pages/EditMonitor.vue

```diff
@@ -745,6 +745,26 @@
                                         {{ $t("acceptedStatusCodesDescription") }}
                                     </div>
                                 </div>
+
+                                <div class="my-3">
+                                    <label for="ipFamily" class="form-label">{{ $t("Ip Family") }}</label>
+
+                                    <select id="ipFamily" v-model="monitor.ipFamily" class="form-select">
+                                        <option :value="null">
+                                            {{ $t("auto-select") }}
+                                        </option>
+                                        <option value="ipv4">
+                                            IPv4
+                                        </option>
+                                        <option value="ipv6">
+                                            IPv6
+                                        </option>
+                                    </select>
+
+                                    <div class="form-text">
+                                        {{ $t("ipFamilyDescription") }} (<a href="https://en.wikipedia.org/wiki/Happy_Eyeballs" target="_blank">{{ $t('Learn More') }}</a>)
+                                    </div>
```

Comment:
A bit of nitting in terms of how the helptext works.
Was a bit wordy and not great for our translators (remember there are LTR and RTL languages) ^^

```suggestion
                                    <i18n-t tag="div" class="form-text" key="ipFamilyDescriptionAutoSelect" v-if="monitor.ipFamily === 'null'">
                                        <template #happyEyeballs>
                                            <a href="https://en.wikipedia.org/wiki/Happy_Eyeballs" target="_blank">"Happy Eyeballs"</a>
                                        </template
                                    </i18n-t>
```